versioned_docs/version-25.10/reporting/installation.md

```diff
@@ -490,18 +567,24 @@ dnf update
 
 ```shell
 dnf install -y dnf-plugins-core
-dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/24.10/el9/centreon-24.10.repo
+dnf config-manager --add-repo https://packages.centreon.com/rpm-standard/25.10/el9/centreon-25.10.repo
 dnf clean all --enablerepo=*
 dnf update
 ```
 
 </TabItem>
 <TabItem value="Debian 12" label="Debian 12">
 
-To install the Centreon repository, execute the following command:
+Install the required packages:
 
 ```shell
-echo "deb https://packages.centreon.com/apt-standard-24.10-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list
+apt install lsb-release ca-certificates apt-transport-https software-properties-common wget gnupg2
+```
+
+Install the Centreon repositories:
+
+```shell
+echo "deb https://packages.centreon.com/apt-standard-25.10-stable/ $(lsb_release -sc) main" | tee /etc/apt/sources.list.d/centreon.list
```

Comment:
```suggestion
echo "deb https://packages.centreon.com/apt-standard/ $(lsb_release -sc)-25.10-stable main" | tee /etc/apt/sources.list.d/centreon.list
```

@kduret  & @tuntoja It's this debian repo now no ?

Reply:
indeed it is, isn t it already added in the "official" 25.10 pull request ?